packages/environment-check/src/checks/service-checks.ts

```diff
@@ -143,7 +143,9 @@ async function catalogRequest(
         if (errorJson?.config?.auth?.password) {
             delete errorJson.config.auth.password;
         }
-        logger.debug(t('error.urlRequestFailure', { url, error: error.message, errorObj: error }));
+        logger.debug(
+            t('error.urlRequestFailure', { url, error: error.message, errorObj: JSON.stringify(errorJson, null, 4) })
```

Comment:
I'm still seeing the error 
`'Converting circular structure to JSON\n    --> starting at object with constructor 'TLSSocket'\n    |     property '_httpMessage' -> object with constructor 'ClientRequest'\n    --- property 'socket' closes the circle'`

To reproduce, any system which only has v2 or v4 (for me EC1 system `https://ccwdfgl9773.devint.net.sap:44300` I only have auth for v2) should throw an error in the catalog request.

Reply:
Hi @cianmSAP, 

I'm still not able to reproduce the issue you are facing, said system has for me both catalog services. However, I did some investigation and found this: https://github.com/axios/axios/issues/4486.

Looks like the issue is solved in `axios@1.x.x`, but unfortunately our module `@sap-ux/axios-extension` uses currently version `axios@0.24.0`.
@tobiasqueck @ullasholla any plans to update `axios` dependency in `@sap-ux/axios-extension`?

For now I'll look into a way to ignore circular structures, like done in [`circular-json`](https://www.npmjs.com/package/circular-json) or  [`json-stringify-safe`](https://www.npmjs.com/package/json-stringify-safe).

Reply:
Updates have resolved the error. Thanks @Klaus-Keller